addon/selection/active-line.js

```diff
@@ -3,9 +3,14 @@
 
 // Because sometimes you need to style the cursor's line.
 //
-// Adds an option 'styleActiveLine' which, when enabled, gives the
-// active line's wrapping <div> the CSS class "CodeMirror-activeline",
-// and gives its background <div> the class "CodeMirror-activeline-background".
+// Adds two options:
+// 'styleActiveLine': when enabled, gives the active line's wrapping
+// <div> the CSS class "CodeMirror-activeline", and gives its background
+// <div> the class "CodeMirror-activeline-background".
+//
+// 'styleActiveSelected': When enabled and 'styleActiveLine' is
+// enabled, keeps the active line's styling active even when text is
+// selected within the line. Has no effect if 'styleActiveLine' is not enabled
```

Comment:
I prefer that you make `styleActiveLine` optionally take an options object as value, instead of a boolean, and read a `nonEmpty` option from there when it is given. That's how other addons tend to do it.